docs/oscar_references.bib

```diff
@@ -386,6 +386,16 @@ @Article{Bis96
   doi           = {10.1112/S0025579300011773}
 }
 
+@Book{Bur55,
+  author        = {Burnside, W.},
+  title         = {Theory of groups of finite order},
+  mrnumber      = {69818},
+  note          = {2d ed},
```

Comment:
Just a guess, but maybe it should be this?
```suggestion
  note          = {2nd ed},
```

Reply:
If depends.
If we want to be consistent with [MathSciNet](https://mathscinet.ams.org/mathscinet/article?mr=69818) then we should keep it as it is.